CONTRIBUTING.md

```diff
@@ -180,7 +181,7 @@ Commits should be signed. You can read more about [Commit Signing][] here.
 
 This project uses [Husky][] for Git pre-commit hooks.
 
-- Some JSON files are generated during Build time with empty files as placeholders. Build time happens when you run `npx turbo serve` or `npx turbo build`.
+- Some JSON files are generated during Build time with empty files as placeholders. Build time happens when you run `pnpm serve` or `pnpm build`.
```

Comment:
also this whole paragraph can be remove. it's outdate because now we use api route instead of custom building static file